bot.py

```diff
@@ -1,63 +1,62 @@
-#!/usr/bin/env python
-
-__author__ = "InfinityLabs"
-__authors__ = ["Infinity"]
-__copyright__ = "Copyright 2013, InfinityLabs"
-__copyright__ = "Copyright 2012, ClouDev"
-__credits__ = ["infinity","thenoodle", "_frozen", "rmmh"]
-__license__ = "GPL v3"
-__version__ = "DEV"
-__maintainer__ = "InfinityLabs"
-__email__ = "root@infinitylabs.us"
-__status__ = "Development"
+#!/usr/bin/env python3
+
+#    _        _             _           _
+#   | |      (_)           | |         | |
+#   | |_ __ _ _  __ _  __ _| |__   ___ | |_
+#   | __/ _` | |/ _` |/ _` | '_ \ / _ \| __|
+#   | || (_| | | (_| | (_| | |_) | (_) | |_
+#    \__\__,_|_|\__, |\__,_|_.__/ \___/ \__|
+#                __/ |
+#               |___/     irc bot
+
+__author__ = 'inexistence'
+__credits__ = ['infinity', 'thenoodle', '_frozen', 'rmmh', '676339784', 'adedomin', 'eskimo', 'blompf']
```

Comment:
![yfw contributed to daddiescummies](https://ih1.redbubble.net/image.2854641226.5367/st,small,845x845-pad,1000x1000,f8f8f8.jpg)

Reply:
sorry about that guints, i added u in https://github.com/inexist3nce/Taigabot/commit/53302b8d5dd08773f75afaed1696f371d079001f

i already tested the bot in irc, i'm currently going through plugins testing every single command but that takes a while cuz many have `u""`/encode/decode hacks to deal with python 2 so i have to undo that before they run